tests/utils/dockercli/volumelifecycle.go

```diff
@@ -42,9 +42,14 @@ func CreateVolumeWithOptions(ip, name, options string) (string, error) {
 // AttachVolume - attach volume to container on given host
 func AttachVolume(ip, volName, containerName string) (string, error) {
 	log.Printf("Attaching volume [%s] on VM [%s]\n", volName, ip)
-	return ssh.InvokeCommand(ip, dockercli.RunContainer+"-d -v "+volName+
-		":/vol1 --name "+containerName+
-		dockercli.TestContainer)
+	return ssh.InvokeCommand(ip, dockercli.RunContainer+" -d --rm -v "+volName+
```

Comment:
How about creating a test util which attach/mounts volume with `--rm` flag to avoid adding cleanup step for such containers removal.